oracle/src/events/processAMBInformationRequests/calls/ethGetStorageAt.js

```diff
@@ -12,7 +14,7 @@ async function callArchive(web3, data, foreignBlock) {
   const { 0: address, 1: slot, 2: blockNumber } = web3.eth.abi.decodeParameters(['address', 'bytes32', 'uint256'], data)
 
   if (toBN(blockNumber).gt(toBN(foreignBlock.number))) {
-    return [false, '0x']
+    return [false, ASYNC_CALL_ERRORS.BLOCK_IS_IN_THE_FUTURE]
```

Comment:
Does it also make sense to handle the situation when there is no storage for an account at all (for EOA accounts)?

Reply:
Ethereum clients usually does not distinguish such cases at all. There is no hardcoded restrictions there, which does not allow for EOA to have non-zero values in its storage. They are simply all-zero values, since there is no way to initialise them.

Also, JSON RPC returns all-zeros value for any eth_getStorageAt call related to EOA, so we are just copying the same behaviour.